InvalidPath.h

```diff
@@ -0,0 +1,27 @@
+#ifndef TICTACTOE_INVALIDPATH_H

```

Comment:
The names of headers should match the namespace / class / struct / etc. it defines.
This name doesn't match and is misleading